react/src/components/atoms/buttons/ButtonWithIcon/style.scss

```diff
@@ -1,10 +1,54 @@
 @import 'global';
+@import '00-base/mixins';
 @import "01-atoms/button-search";
 
+$colors: (
+  "c-highlight": $c-highlight,
+  "c-primary": $c-primary,
+  "c-primary-alt": $c-primary-alt,
+  "c-black": $c-black,
+  "c-gray-dark": $c-gray-dark,
+  "c-white": $c-white
+);
+@mixin ma-button-icon-secondary ($color: $c-primary-alt) {
+  svg {
+    fill: $color;
+  }
+  &:hover {
+    svg {
+      fill: $c-white;
+    }
+  }
+}
+@mixin ma-button-icon-quaternary ($color: $c-primary-alt) {
+  border-bottom-color: rgba($color,0.75);
+  color: $color;
+  svg {
+    fill: rgba($color,0.75);
+  }
+  &:hover {
+    border-bottom-color: $color;
+  }
+}
+@mixin ma-button-icon-quaternary-simple ($color: $c-primary-alt) {
```

Comment:
In regular button component, quaternary is hard coded to only render theme `$c-primary` (all the other theme colors don't work) 
I think we should extend the buttons to allow button quarternary to use more themes except for hightlight, as currently they don't work in all quarternary usage for accessiblity
<img width="136" alt="Screen Shot 2019-09-06 at 3 55 15 PM" src="https://user-images.githubusercontent.com/5789411/64457236-12d8b100-d0c0-11e9-81cc-bb63743d94c8.png">
<img width="146" alt="Screen Shot 2019-09-06 at 3 55 18 PM" src="https://user-images.githubusercontent.com/5789411/64457237-13714780-d0c0-11e9-80a0-13bfa61c7033.png">

Reply:
removed option for quaternary to use c-highlight theme